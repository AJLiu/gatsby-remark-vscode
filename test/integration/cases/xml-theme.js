module.exports = {
  colorTheme: `Oceanic Next`,
  extensions: [
    {
      identifier: `naumovs.theme-oceanicnext`,
      version: `0.0.4`,
    },
  ],
};
