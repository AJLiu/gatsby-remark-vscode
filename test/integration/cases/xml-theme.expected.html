<pre class="grvsc-container oceanic-next" data-language="js"><code class="grvsc-code"><span class="grvsc-line"><span class="mtk7">console</span><span class="mtk5">.</span><span class="mtk8">log</span><span class="mtk1">(</span><span class="mtk5">"</span><span class="mtk6">Hello, world!</span><span class="mtk5">"</span><span class="mtk1">)</span><span class="mtk5">;</span></span></code></pre>
<style class="grvsc-styles">
  :root {
    --grvsc-padding-v: 1rem;
    --grvsc-padding-h: 1.5rem;
    --grvsc-padding-top: var(--grvsc-padding-v);
    --grvsc-padding-right: var(--grvsc-padding-h);
    --grvsc-padding-bottom: var(--grvsc-padding-v);
    --grvsc-padding-left: var(--grvsc-padding-h);
    --grvsc-border-radius: 8px;
  
    --grvsc-line-highlighted-background-color: transparent;
    --grvsc-line-highlighted-border-width: 4px;
    --grvsc-line-highlighted-border-color: transparent;
  }
  
  .grvsc-container {
    overflow: auto;
    -webkit-overflow-scrolling: touch;
    padding-top: 1rem;
    padding-top: var(--grvsc-padding-top);
    padding-bottom: 1rem;
    padding-bottom: var(--grvsc-padding-bottom);
    border-radius: 8px;
    border-radius: var(--grvsc-border-radius);
    font-feature-settings: normal;
  }
  
  .grvsc-code {
    display: inline-block;
    min-width: 100%;
  }
  
  .grvsc-line {
    display: inline-block;
    box-sizing: border-box;
    width: 100%;
    padding-left: 1.5rem;
    padding-left: var(--grvsc-padding-left);
    padding-right: 1.5rem;
    padding-right: var(--grvsc-padding-right);
  }
  
  .grvsc-highlighted {
    background-color: var(--grvsc-line-highlighted-background-color);
    box-shadow: inset var(--grvsc-line-highlighted-border-width) 0 0 0 var(--grvsc-line-highlighted-border-color);
  }
  
  .oceanic-next {
    background-color: #1B2B34;
    color: #CDD3DE;
  }
  .oceanic-next .mtk7 { color: #FAC863; }
  .oceanic-next .mtk5 { color: #5FB3B3; }
  .oceanic-next .mtk8 { color: #6699CC; }
  .oceanic-next .mtk1 { color: #CDD3DE; }
  .oceanic-next .mtk6 { color: #99C794; }
</style>